nvme_gw_cli.py

```diff
@@ -29,14 +29,24 @@ class Parser:
         parser: ArgumentParser object.
         subparsers: Action object to add subcommands to main argument parser.
     """
+
     def __init__(self):
         self.parser = argparse.ArgumentParser(
-            prog='gwcli',
-            description='Manage NVMe gateways')
+            prog="gwcli", description="Manage NVMe gateways"
+        )
+        # self.parser.add_argument("config_file", help="Gateway configuration file")
+        self.parser = argparse.ArgumentParser(
+            prog="nvme_gw_cli", description="CLI to manage NVMe gateways"
+        )
         self.parser.add_argument(
-            'config_file', 
-            help='Gateway configuration file')
-        self.subparsers = self.parser.add_subparsers(dest='subcommand')
+            "-c",
+            "--config-file",
+            default="./nvme_gw.config",
```

Comment:
nit: wouldn't `default="nvme_gw.config"` be the same? It looks a bit nicer.

Reply:
ok